### PR TITLE
Add config file and content (WIP)

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add config file 
Add some content to index (WIP)
Use vanilla brochure theme (WIP)
Update gems

## QA

Follow the instructions if README.mp to get up and running and check that there the data in the config file is shown on the home page.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely need to edit after that.
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+title: conjure-up
+email: webteam@canonical.com
+description: Deployments so easy it's almost magical.
+keywords: snappy, Ubuntu Core, Ubuntu 16.04 LTS, snaps, apps, debs, snap, docker, coreos, atomic, snapcraft, snapd
```
Comment (anthonydillon) [inline _config.yml]: These seem like snap keywords.

Comment (anthonydillon) [inline _config.yml]: I also think we could probably drop meta keywords these days.

File: _layouts/base.html [hunk 2]
```diff
@@ -45,7 +43,7 @@
                 </span>
                 <div class="logo">
                     <a href="/">
-                        <img src="" alt="conjure-up.io">
+                        <span>conjure-up.io</span>
```
Comment (anthonydillon) [inline _layouts/base.html]: Wouldn't the site be titled `conjure-up` not including the `.io`.

File: index.html [hunk 3]
```diff
@@ -2,4 +2,10 @@
 layout: default
 ---
 
-Hello world!
+<h1>Hello world</h1>
+
+<p class="intro">Deployments so easy it's almost magical</p>
```
Comment (anthonydillon) [inline index.html]: There is no `.intro` in Vanilla you could use the [heading class](https://docs.vanillaframework.io/en/patterns/headings).

File: index.html [hunk 4]
```diff
@@ -2,4 +2,10 @@
...
+
+<h2>Conjure-up is a power tool for getting started with big software</h2>
+
+<p>Conjure-up lets you summon up a big-software stack as a “spell” - a model of the stack, combined with extra know-how to get you from an installed stack to a fully usable one. Start using your big software instead of learning how to deploy it.</p>
```
Comment (anthonydillon) [inline index.html]: Could you please encode the special characters

Comment (grahambancroft) [inline index.html]: Fixed but has changed in another branch anyway

File: index.html (hunk 3 above)
Comment (grahambancroft) [inline index.html]: Fixed but has changed in another branch anyway

File: _layouts/base.html (hunk 2 above)
Comment (grahambancroft) [inline _layouts/base.html]: Fixed but has changed in another branch anyway

File: _config.yml (hunk 1 above)
Comment (grahambancroft) [inline _config.yml]: Done

**Comment by grahambancroft** [comment]
@anthonydillon Travis checks are failing because of a path issue in vanilla-brochure-theme, I need to investigate this further

**Comment by anthonydillon** [comment]
I'm doing a new release of brochure theme because I fixed the issue but it hasnt been released.

**Comment by anthonydillon** [comment]
Code is good but need to fix CI

**Comment by grahambancroft** [comment]
[Move site to Vanilla](https://trello.com/c/oyXjUgMs/2757-move-site-to-vanilla)

File: package.json [hunk 5]
```diff
@@ -44,5 +45,8 @@
     "node-sass": "^4.5.0",
     "sass-lint": "^1.10.2",
     "uglify-js": "^2.8.10"
+  },
+  "dependencies": {
+    "vanilla-brochure-theme": "0.0.3"
```
Comment (anthonydillon) [inline package.json]: Please update to `1.0.1`

**Comment by anthonydillon** [comment]
@grahambancroft You will still need to update your includePaths ^

**Pull request merged**